vars/helm.groovy

```diff
@@ -16,10 +16,40 @@ def undeployChart(environment, chartName, tag) {
   Helm.undeployChart(this, environment, chartName, tag)
 }
 
-def publishChart(registry, chartName, tag) {
-  Helm.publishChart(this, registry, chartName, tag)
+def publishChart(registry, chartName, tag, helmChartLocation="artifactory") {
+  if (helmChartLocation) {
+    def location = helmChartLocation.toLowerCase()
+
+    if (location == 'acr') {
```

Comment:
would it be better to have artifactory as the first condition check if the majority of services are going to use it to save string matching against acr for no reason.

Reply:
Done. Changed to switch statement too.